book/src/merge-migration.md

```diff
@@ -1,20 +1,69 @@
 # Merge Migration
 
-This document provides detail for users who have been running a Lighthouse node *before* the merge
-and are now preparing their node for the merge transition.
+This document provides detail for users who want to run a merge-ready Lighthouse node.
 
-## "Pre-Merge" and "Post-Merge"
+> If you are running a testnet node, this configuration is necessary _now_.
 
-As of [v2.4.0](https://github.com/sigp/lighthouse/releases/tag/v2.4.0) Lighthouse can be considered
-to have two modes:
+## Necessary Configuration
 
-- "Pre-merge": `--execution-endpoint` flag *is not* provided.
-- "Post-merge": `--execution-endpoint` flag *is* provided.
+There are two configuration changes required for a Lighthouse node to operate correctly throughout
+the merge:
 
-A "pre-merge" node, by definition, will fail to transition through the merge. Such a node *must* be
-upgraded before the Bellatrix upgrade.
+1. You *must* run a local execution engine such as Geth or Nethermind alongside Lighthouse.
+   You *must* update your Lighthouse configuration to connect to the execution engine using new
+   flags which are documented on this page in the
+   [Connecting to an execution engine](#connecting-to-an-execution-engine) section.
+2. If your Lighthouse node has validators attached you *must* nominate an Ethereum address to
+   receive transactions tips from blocks proposed by your validators. This is covered on the
+   [Suggested fee recipient](./suggested-fee-recipient.md) page.
 
-## Migration
+Additionally, you _must_ update Lighthouse to a merge-compatible release in the weeks before
+the merge. Merge releases are available now for all testnets.
+
+## When?
+
+You must configure your node to be merge-ready before the Bellatrix fork occurs on the network
+on which your node is operating.
+
+* **Mainnet**: the Bellatrix fork epoch has not yet been announced. It's possible to set up a
+  merge-ready node now, but some execution engines will require additional configuration. Please see
+  the secton on [Execution engine configuration](#execution-engine-configuration) below.
+
+* **Goerli (Prater)**, **Ropsten**, **Sepolia**, **Kiln**: you must have a merge-ready configuration
```

Comment:
Are these all ready "post-merge"? I know goerli/prater is :)

Reply:
Yep! Goerli is actually only post-Bellatrix, but hasn't merged with the execution chain. The rest have completed the merge